src/serialization/src/main/kotlin/org/vaccineimpact/api/serialization/ModelBinder.kt

```diff
@@ -33,15 +34,12 @@ class ModelBinder(private val serializer: Serializer = MontaguSerializer.instanc
         @Suppress("UNCHECKED_CAST")
         val klass = model::class as KClass<Any>
         val errors = mutableListOf<ErrorInfo>()
-        val name = property.name
+        val name = serializer.convertFieldName(property.name)
         val value = property.get(model)
 
-        if (value == null && !property.returnType.isMarkedNullable)
+        if (!property.returnType.isMarkedNullable)
         {
-            errors += ErrorInfo(
-                    "invalid-field:$name:missing",
-                    "You have not supplied a value, or have supplied a null value, for field '$name'"
-            )
+            errors += listOfNotNull(checkMissing(value, name))
```

Comment:
This change is so we can share the "error on null" logic with the new `RequiredWhen` annotation.